scene/gui/text_edit.cpp

```diff
@@ -2065,6 +2065,12 @@ void TextEdit::_input_event(const InputEvent& p_input_event) {
 
 					if (k.mod.shift)
 						_pre_shift_selection();
+#ifdef APPLE_STYLE_KEYS
+					else
```

Comment:
Oops... yeah, `deselect()` was supposed to be outside that preprocessor block... What a miss!